devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -34,6 +34,7 @@ metadata {
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05,FC02", outClusters: "0019", manufacturer: "CentraLite", model: "3321-S", deviceJoinName: "Multipurpose Sensor"
 		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500,FC02", outClusters: "0019", manufacturer: "SmartThings", model: "multiv4", deviceJoinName: "Multipurpose Sensor"
 		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,FC02", outClusters: "0019", manufacturer: "Samjin", model: "multi", deviceJoinName: "Multipurpose Sensor"
+		fingerprint profileId: "0104,0402", inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0003,0019", manufacturer: "Sercomm Corp.", model: "SZ-DWS04", deviceJoinName: "Door Window Sensor"
```

Comment:
The `profileId` is not required but if present it should only have one value which is typically `0104`

Reply:
Please double check the `deviceJoinName`. According to the ticket it should include `Sercomm`.